### DISCUSSION TITLE
Request for Information on Release Process and Lazy Evaluation Feature

### DISCUSSION THREAD

**Discussion by Pozo** [ezylang/EvalEx#456] [Q&A] [answered]
First of all, big thanks for all the hard work you've put into developing and maintaining this library!

Currently, I'm in the process of evaluating various alternatives to [mXparser](https://mathparser.org/), and it appears that EvalEx aligns more closely with our objectives compared to other options. However, we do require the lazy evaluation feature, which I noticed has been [implemented](https://github.com/ezylang/EvalEx/issues/427) but not yet released.

I've taken the time to review the documentation and the lisf of issues, but unfortunately, I couldn't find any information regarding your release process. Would it be possible for you to provide insight into your current release process and perhaps an estimate of when we can expect a version that includes the aforementioned feature?

**Comment by uklimaschewski** [comment, marked as answer]
Hi,

Thank you and I have good news for you: [EvalEx version 3.2.0 ](https://github.com/ezylang/EvalEx/discussions/460) has been released today.

**Reply by Pozo** [reply]
Hi @uklimaschewski, That's good news! Thank you for letting me know ! I'll jump on it right away.
